imports/ui/pages/currencyDetail/features.js

```diff
@@ -1,131 +1,170 @@
-import { Template } from 'meteor/templating'
-import { HashAlgorithm, Currencies, Features } from '/imports/api/indexDB.js'
+import {
+    Template
+} from 'meteor/templating'
+import {
+    HashAlgorithm,
+    Currencies,
+    Features
+} from '/imports/api/indexDB.js'
 import Cookies from 'js-cookie'
 
 import './features.html'
+import { FeatureValidate } from "../../../validate/featuresValidate";
 
-Template.features.onCreated(function(){
-  this.showflagged = new ReactiveVar(false)
-  this.addingnewfeature = new ReactiveVar(false)
-  this.lastId = new ReactiveVar('')
-  this.featureIncrement = 3
-  this.featureLimit = new ReactiveVar(this.featureIncrement)
-  this.featuresShow = new ReactiveVar(null)
-  this.autorun(() => {
-    SubsCache.subscribe('featuresSlug', FlowRouter.getParam('slug'))
-  })
 
-  //prefetch next load of features into minimongo
-  this.autorun(()=>{
-    var query = {currencySlug: FlowRouter.getParam('slug'), flagRatio: {$lt: 0.6}}
-    this.featuresShow.set(Features.find(query, {limit: this.featureLimit.get(), sort: {rating: -1, appealNumber: -1,createdAt:-1}}));
-    //prefetch
-    Features.find(query, {limit: this.featureLimit.get()+this.featureIncrement, sort: {rating: -1,createdAt:-1, appealNumber: -1}}).fetch()
-  })
+Template.features.onCreated(function() {
+    this.showflagged = new ReactiveVar(false)
```

Comment:
@jigartankhupp Can we use one tab indentation?

Reply:
Sure we should. I'll do it. But right now this indentation needs to be done throughout the entire code base. @geekyd 
So that will come as complete separate issue and PR for it.

Reply:
@geekyd sure. updating.

Reply:
@geekyd done with indentation.